spec/controllers/apps_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'spec_helper'
+
+# rubocop:disable Metrics/BlockLength
+RSpec.describe AppsController do
+  let(:example_app) { FactoryBot.create(:app, :with_event_type) }
+  setup do
+    example_app.save!
```

Comment:
`example_app` will still need to be called, for the app to be in the database when the tests run, as those don't call it themselves. `save!` might be redundant, but is helpful to ensure that data is there. In case the factory at some point produces invalid entries, `.create` would not save or raise an error, while tests would without an apparent reason fail.